src/components/timers/Tabata.tsx

```diff
@@ -1,3 +1,159 @@
-const Tabata = () => null;
+import { useEffect, useState } from 'react';
+import ActionButton from '../generic/ActionButton';
+import DisplayWindow from '../generic/DisplayWindow';
+import InputField from '../generic/Input';
+import Loading from '../generic/Loading';
+
+//Manages state
+const Tabata = () => {
```

Comment:
Tabata has a lot of different states, but you're not displaying much to the user. How do I know what round I'm in? how do I know if I'm in a work or rest period?